### DISCUSSION TITLE
Proximity Sensor and light Sensor Error

### DISCUSSION THREAD

**Discussion by JuergenVoelkl** [seaky/nspanel_pro_tools_apk#88] [Q&A]
Hi, I have a problem with the Proximity Sensor and light Sensor on your App, do you have any idea why this happens?
Thanks a lot!!!

**Comment by JuergenVoelkl** [comment]
![Untitled](https://github.com/seaky/nspanel_pro_tools_apk/assets/167901733/e3ed4e58-0401-4ba8-9ee6-086fe7c8e8b6)

**Comment by seaky** [comment]
Hi,

Please on settings page enable debug and verbose log after that switch back to sensor tab and attach or paste here the log file.

**Reply by JuergenVoelkl** [reply]
Thanks for the quick answear, I can get along until debug and verbose log are activated, but I don't know how to attach or paste the log file.

**Reply by JuergenVoelkl** [reply]
![NS](https://github.com/seaky/nspanel_pro_tools_apk/assets/167901733/39b9d564-2381-436c-9e45-46171805e47a)
